SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1068,7 +1068,7 @@ void USpatialSender::UpdateInterestComponent(AActor* Actor)
 	}
 
 	InterestFactory InterestUpdateFactory(Actor, ClassInfoManager->GetOrCreateClassInfoByObject(Actor), NetDriver->ClassInfoManager, NetDriver->PackageMap);
-	Worker_ComponentUpdate Update = InterestUpdateFactory.CreateInterestUpdate();
+	Worker_ComponentUpdate Update = InterestUpdateFactory.CreateInterestUpdate(EntityId);
```

Comment:
Should entity id just be passed in to the factory constructor?

Reply:
I really want to move away from the pattern of a constructor that is called once, used, then discarded. I am only passing the entity ID here as modifying the factory to be more persistent is a bigger change, and not required as part of this one, but the idea is that eventually all arguments will be passed to the call in order to be explicit

Reply:
It is the pattern of the factory though, and we use it elsewhere, so I'm not sure I see the benefit of not being consistent with it's usage.

Reply:
sure, changed